### PR TITLE
Sanitise HTML for Jekyll

### LINKED ISSUE #46: Fix the HTML Jekyll is choking on
In numerous places, Jekyll chokes on HTML formatting. Things like "<br clear=\"all\" />" make Jekyll print a frightening and annoying error message on every compile.

They should be hunted down and eliminated with extreme prejudice.

### PR DESCRIPTION
**Pull request by paddycarver**
I reformatted all the HTML that Jekyll would stop whining every time we ran it locally.

Fun facts:
1. Try to keep your HTML within HTML blocks. A lot of these errors were just dangling `<br />` tags. Some tags were used as clear elements after floating text right. `width: 100%; text-align: right` works just as well (correct me if I'm wrong!) and lets us get away without, so let's use that.
2. Copying and pasting the raw HTML from Google Docs or Google Sites makes me cry. Take pride in your formatting!

This request would fix #46.

### CODE REVIEW

**Comment by carimura** [comment]
What do you mean dangling br /?

And how do we handle clearing a float without a clear=all? I noticed that error but ignored it because that's how it's cleared.

**Comment by paddycarver** [comment]
By dangling `<br />`, I just mean a `<br />` that is outside of any block-level element.

**Dangling**:

```
<p>My awesome text</p>
<br />
```

**Not Dangling**

```
<p>My awesome text.<br /></p>
```

As for the question of formatting, the fix I came up with is used in this pull request. Instead of `<p><a style="float: right;" href="..">text</a></p>`, which needs the clear, I use `<p style="width: 100%; text-align: right;"><a href="...">text</a></p>`, which is visually the same but doesn't require the clear.

**TL;DR**: Instead of floating things, set width to 100% and text-align them instead. It makes my OCD hurt less and makes the error output of Jekyll more useful.
